internal/provider/resource_value.go

```diff
@@ -125,19 +125,16 @@ func (r *ResourceValue) Schema(ctx context.Context, req resource.SchemaRequest,
 						MarkdownDescription: "Secret reference in the format of the target store. It can't be defined if value is defined.",
 						Optional:            true,
 						Computed:            true,
-						PlanModifiers:       []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
 					},
 					"store": schema.StringAttribute{
 						MarkdownDescription: "Secret Store id. This can't be humanitec (our internal Secret Store). It's mandatory if ref is defined and can't be used if value is defined.",
 						Optional:            true,
 						Computed:            true,
-						PlanModifiers:       []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
 					},
 					"version": schema.StringAttribute{
 						MarkdownDescription: "Only valid if ref is defined. It's the version of the secret as defined in the target store.",
 						Optional:            true,
 						Computed:            true,
-						PlanModifiers:       []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
```

Comment:
We could actually keep the plan modifier for `store` and `ref` as they shouldn't change (if not `null`) but it's not the case for `version`.